cypress/integration/pageobjects/account-management/roles-page.ts

```diff
@@ -0,0 +1,65 @@
+import { randomString } from '../../../support/commands';
+
+export default class RolesPage{
+    selectRolesTab() {
+        getMainIframeDocumentWaitToLoad().find('ol.nav-tabs  > li > a', {timeout: 30000}).contains('Roles').should('exist').click();
```

Comment:
You can replace getMainIframeDocumentWaitToLoad and getMainIframeDocument with the common method getIframeBody() from command.ts, also apply the changes in users-page.ts